hub/client/hub_control.py

```diff
@@ -66,6 +67,24 @@ def get_credentials(self):
         self.save_config(details)
         return details
 
+    def get_dataset_credentials(self, org_id, ds_name):
+        self.auth_header = (
+            self.auth_header
+            if self.auth_header is not None
+            else f'Bearer {AuthClient().get_access_token(username="public", password="")}'
+        )
+        relative_url = config.GET_DATASET_CREDENTIALS_SUFFIX
+        relative_url = relative_url.split("/")
+        relative_url[3] = org_id
+        relative_url[5] = ds_name
+        relative_url = "/".join(relative_url)
```

Comment:
I got what this part does, but someone unfamiliar might be confused as to what is happening here.

Reply:
Maybe use C-style formatting?